### PR TITLE
Create routers for each table

### PR DESCRIPTION
**Pull request by davidhurtadop**
Routers for each table

### CODE REVIEW

**Review by calbertora — COMMENTED** [review]
Please fix the comments

File: index.js [hunk 1]
```diff
@@ -1,49 +1,17 @@
...
-app.get('/', (req, res) => {
-    res.send('hello from simple server :)')
-})
 
-app.get('/users', async(req, res) => {
-    pool.query('SELECT * FROM public."Users"', async(errs, resu) => {
-        console.log(errs, resu.rows)
-        res.send(resu.rows)
-    })
-})
```
[the remark this answers is not in the document]
Comment (davidhurtadop) [reply index.js]: Fixed

File: routers/users.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+})
+
+router.post('/', async(req, res) => {
+    pool.query('INSERT INTO public."Users" ("Id","Full_Name","login","password") VALUES ($1,$2,$3,$4)', [4, "Mia", null, null], async(errs, resu) => {
+        console.log(errs, resu)
+        res.send('User Added Succesfully')
+    })
+})
+
+
```
[the remark this answers is not in the document]
Comment (davidhurtadop) [reply routers/users.js]: Fixed

**Comment by calbertora** [comment]
LGTM 👍

**Review by calbertora — APPROVED** [review]
LGTM 👍

**Pull request merged**